vm-setup/roles/v1aX_integration_test/templates/clusterctl_env_template_centos.rc

```diff
@@ -108,7 +109,7 @@ export CTLPLANE_KUBEADM_EXTRA_CONFIG="
           }
           vrrp_instance VI_1 {
               state MASTER
-              interface eth1
+              interface {% if EXTERNAL_VLAN_ID == "" %}eth1{% else %}eth0.{{ EXTERNAL_VLAN_ID }}{% endif %}
```

Comment:
I would expect something similar for Ubuntu, no ?

Reply:
Good spot, thanks. I've added the keepalived config for the Ubuntu VLAN interface too.